app/models/form.rb

```diff
@@ -223,4 +223,12 @@ def depends_on_met(depends_on, log)
       end
     end
   end
+
+  def in_crossover_period?(now: Time.zone.now)
+    ((end_date - 3.months) < now) && (now < end_date)
```

Comment:
I'd say we could use start dates of the forms to determine this instead of subtracting 3 months.
We could potentially get all the `lettings_forms` or `sales_forms` from form handler and check if today is between start_date and end_date for more than one of them.
Maybe this belongs in FormHandler instead of the model too?

Reply:
I see, yeah I wasn't aware they overlapped that way. Let me have a go and I'll get back you.

Reply:
I've done this in another PR https://github.com/communitiesuk/submit-social-housing-lettings-and-sales-data/pull/1043 the commit is https://github.com/communitiesuk/submit-social-housing-lettings-and-sales-data/pull/1043/commits/0528cd572137c1b59474b08766883f16f1d84c26